packages/provider_approach/lib/features/profile/edit/edit_profile_delegate.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_firebase_auth_blueprint/features/profile/preview/profile_widget.dart';
+
+// ignore: one_member_abstracts
```

Comment:
It does not work for abstract classes which contains 2 or more methods. Please remove it.